<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMOtherPhoto</class>
 <widget class="QWidget" name="SMOtherPhoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>840</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="tools" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QToolButton" name="tbDaochu">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>4</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShanchu">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>4</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbTianjia">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>4</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAllSelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSelectCount">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>7</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已选0项</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShaxin">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>4</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
